crypto/commitment/hash.go

```diff
@@ -50,30 +49,24 @@ func NewHashCommitmenter(data []byte, minSaltSize int) (*HashCommitmenter, error
 	if lens < minSaltSize {
 		lens = minSaltSize
 	}
-	randomSalt, err := utils.GenRandomBytes(lens)
```

Comment:
minSaltSize should be removed, because the bit-length of the salt always 32.

Reply:
updated